### PR TITLE
Crate a standalone Activity Page

### PR DESCRIPTION
**Pull request by KyleJu**
#2831. Create a standalone activity page that shows all activities and comments associated with a feature.

### CODE REVIEW

**Comment by KyleJu** [comment]
A screenshot
Screenshot 2023-04-19 at 3 16 26 PM

File: client-src/elements/chromedash-activity-page.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  }
+
+  renderComments() {
+    // TODO(jrobbins): Include relevant activities too.
+    return html`
+      <h2>Comments</h2>
+      ${this.renderControls()}
+      <chromedash-activity-log
+        .user=${this.user}
+        .feature=${this.featureId}
```
Comment (jrobbins) [inline client-src/elements/chromedash-activity-page.js:156]: Isn't the property name now `featureId`?

**Review by jrobbins — CHANGES_REQUESTED** [review]

File: main.py [hunk 2]
```diff
@@ -151,6 +151,7 @@ class Route:
   Route('/myfeatures', defaults={'require_signin': True}),
   Route('/newfeatures'),
   Route('/feature/<int:feature_id>'),
+  Route('/activity/<int:feature_id>'),
```
[the remark this answers is not in the document]
Comment (KyleJu) [reply main.py:154]: That is a better path!

File: client-src/elements/chromedash-feature-detail.js [hunk 3]
```diff
@@ -616,7 +616,7 @@ class ChromedashFeatureDetail extends LitElement {
         <div style="padding-top: var(--content-padding)">
           <chromedash-activity-log
             .user=${this.user}
-            .feature=${this.feature}
+            .feature=${this.feature.id}
```
[the remark this answers is not in the document]
Comment (KyleJu) [reply client-src/elements/chromedash-feature-detail.js]: Yup I will send a follow-up PR

**Review by jrobbins — APPROVED** [review]

**Pull request merged**